tests/datalab/issue_manager/test_duplicate.py

```diff
@@ -86,6 +86,17 @@ def test_find_issues(self, issue_manager, embeddings):
         )
         new_issue_manager.find_issues(features=embeddings["embedding"])
 
+    def test_scores_of_examples_with_issues_are_smaller_than_those_without(
+        self, issue_manager, embeddings
+    ):
+        # TODO: Turn this into a property-based test
```

Comment:
This is more of a reminder to make this a more robust test.
It's not strictly related to the PR.
But since we're changing how the score looks, it may be a good idea to have this sanity-check.

I may open an issue to track this todo. It should be addressed in a follow-up PR.

Reply:
opening GH issue sounds good!